package.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>sustainml_frontend</name>
```

Comment:
I'd rename it to `sustainml_dashboard` or just `sustainml`. I'd prefer just `sustainml`

```suggestion
  <name>sustainml_dashboard</name>
```